packages/search/lib/indices/repos.js

```diff
@@ -0,0 +1,13 @@
+const type = 'Repo'
+
+module.exports = {
+  type,
+  name: type.toLowerCase(),
+  searchable: false,
```

Comment:
Odd naming. Maybe just explicitly define which index you want to search in `search/graphql/resolvers/_queries/search.js` or expose a default list in `search/lib/indices/index.js` without repos?

Reply:
Understand, yet sticking to it.